include/zephyr/debug/coredump.h

```diff
@@ -11,6 +11,14 @@
 #include <stdint.h>
 #include <sys/types.h>
 
+/**
```

Comment:
Use a single `*` or else this would be considered a documentation for `COREDUMP_BEGIN_STR`.

Reply:
Done.